typescript-getting-started/functions/src/index.ts

```diff
@@ -0,0 +1,12 @@
+import * as functions from 'firebase-functions'
+
+// if you need to use the Firebase Admin SDK, uncomment the following:
+// import * as admin from 'firebase-admin'
+
+
+// // Create and Deploy Your First Cloud Functions
```

Comment:
Double comments are weird here. I'd probably either leave the whole function commented or remove the double comments & empty comment line.